test/e2e_node/jenkins/docker_validation/docker-validation-config.yaml

```diff
@@ -0,0 +1,5 @@
+images:
```

Comment:
rename this to validation-config.yaml since the file is already under docker_validation?